code/go/pkg/validator/validator_test.go

```diff
@@ -59,19 +59,21 @@ func TestValidateItemNotAllowed(t *testing.T) {
 		"wrong_kibana_filename": {
 			"kibana/dashboard": []string{
 				"b7e55b73-97cc-44fd-8555-d01b7e13e70d.json",
-				"bad-dashboard.json",
+				"bad-ecs.json",
+				"bad-foobar-ecs.json",
+				"bad-Foobaz-ECS.json",
 			},
 			"kibana/map": []string{
 				"06149856-cbc1-4988-a93a-815915c4408e.json",
-				"bad-map.json",
+				"not-bad-map.json",
```

Comment:
Sorry, I'm a little confused about this name change. What was the problem if you left it as `bad-map.json`?

Reply:
If you take a look at `test/package`, the name `good` is restricted to the package which is 100% aligned to the schema.   This package here is not good - `bad` :) which means that `bad-map.json` is a valid Kibana file, but I need a one with wrong name here.

Reply:
Ah, got it, thanks for explaining. `not-bad` is quite a twisted way of saying `good` so how about we just give these files UUIDs as names? Anyway I don't see any difference between this file and the one above it (`06149856-cbc1-4988-a93a-815915c4408e.json`) - they are both files with empty JSON objects in them and neither of them have a `-ecs` suffix in the name. Or am I missing something again? 😄

Reply:
The intention was:
* `06149856-cbc1-4988-a93a-815915c4408e.json` is a wrong file without package name prefixed.
* `not-bad-map.json` is a file with a different package name.

I suppose I can fallback to the first option only (keep the `06149856-cbc1-4988-a93a-815915c4408e.json` file only).

Reply:
I see. Thanks for clarifying. I think the issue is that it's not obvious from the filenames what purpose they are meant to serve in the tests. I would suggest keeping two files, one to test each case that you explained. But:
- maybe instead of calling it `not-bad-map`, call it something like `another-package-map`, only because `not-bad` is confusing ("why didn't they just call it `good`", someone looking at this file might ask), and
- maybe flesh out each of the files just enough to get a description in there, where we can see what purpose that file is serving in tests.

Reply:
> maybe instead of calling it not-bad-map, call it something like another-package-map, only because not-bad is confusing ("why didn't they just call it good", someone looking at this file might ask), and

`another-package-map.json` LGTM , I will adjust the code.

> maybe flesh out each of the files just enough to get a description in there, where we can see what purpose that file is serving in tests.

I can add two properties, ID and description, to add more context to the test case. I wouldn't pollute it with a real Kibana object with all properties as I believe it's not necessary them.

Reply:
> I can add two properties, ID and description, to add more context to the test case. I wouldn't pollute it with a real Kibana object with all properties as I believe it's not necessary them.

👍  I think it might be good to follow a real Kibana object's schema though. That way if we ever start validating Kibana object structures, we don't suddenly start failing this test. But ++ to only populating the minimum number of properties needed.

Reply:
Done